Wiki/models/models-aus2200.md

```diff
@@ -0,0 +1,171 @@
+
+The runtime for this task is variable, if all grib files are already present, then it completes in a few seconds. If none of the grib files are present, this task can take from 10 minutes to almost an hour. Unfortunately the cause of this variability is not known. 
+
+The configuration settings for this task and for `get_lbc_from_archive` can be found in the 'Driving model setup' section of the rose config editor
+![driving model setup](../Images/AUS2200/driving-model-setup.png)
+
+### `ec_recon_nnn`
+This is the primary driving model reconfiguration task. It creates output files on two domains, the initial AUS2200 domain, and a slightly larger domain of the same resolution that acts as the spatial boundary conditions as the model progresses. One of these tasks is run for each point in the model at which the boundary conditions are synchronised, including the initial and final timesteps. In the default configuration, this means seven tasks per 6 hour cycle. These tasks are not run if the `get_lbc_from_archive` task in the same cycle completed successfully. The exception to this is in the inital cycle, where `ec_recon_000` is run in order to create the initial dump file for the model run.
+
+By default these tasks are configured to run on 2 Sapphire Rapids nodes (`normalsr` queue, 208 cpus) each and complete in approximately 38 minutes. The tasks are independent, and are configured to run in parallel. This block of tasks is the second most expensive task set of the workflow, at a cost of about 1.8kSU/cycle. The output is saved to `{{CYLC_SUITE_SHARE_DIR}}/cycle/{{CYLC_TASK_CYCLE_POINT}}/ec/um`, and can be cleared after the forecast task has been completed.
+
+### `aus2200_d0198_RA3_um_createbc_000`
+This task takes the driving model reconfiguration output and creates the boundary 'frames' that are read by the model every hour to synchronise along the spatial boundaries. These frames are fairly small and data-derived, therefore are ideal for re-use in the 'restart' run mode where spin-up times and model runtimes overlap. This task is not run if the `get_lbc_from_archive` task in the same cycle was successful. The task takes approximately 5 minutes to complete on a single core.
+
+### `put_lbc_to_archive`
+This task archives the boundary conditions created by `aus2200_d0198_RA3_um_createbc_000` to the directory determined above in the `get_lbc_from_archive` task. Note that there is a race condition between testing whether archived boundary conditions are available, to this task running. When running overlapping restart suites, it is best to ensure that the same cycle point is not running in two different suites to avoid this race condition. This task is not run if `Archive LBCS` is set to false in the 'Driving model setup' section of the rose suite configuration (`{{ARCHIVE_LBCS}}` in `rose-suite.conf`)
```

Comment:
I find this part of the sentence confusing: "to the directory determined above in the `get_lbc_from_archive` task" as there's no reference to a directory being some kind of output in the get_lbc_from_archive section. Also maybe use "previously" instead of "above". You can probably cut it out as it's kind of obvious that the boundary conditions will have to be store somewhere, or mention the directory in the get_lbc_.. if this is an important detail

Reply:
Also add a note somewhere of what a "race condition" is (even just a link to wikipedia?) as it won't be obvious for many researchers. And again I would put "This task is not run ..." before the Note on the race condition